electron/electron.js

```diff
@@ -13,12 +14,15 @@ if (squirrel) {
   app.quit();
 }
 
+// declare mainWindow ahead of time so that it can be accessed in .handle('get-file')
+let mainWindow;
 const createWindow = () => {
   // Create the browser window.
-  const mainWindow = new BrowserWindow({
+  mainWindow = new BrowserWindow({
     width: 800,
     height: 600,
     webPreferences: {
+      enableRemoteModule: true,
```

Comment:
Is there a reason this was added? I believe that the api/preload style that we've set up avoids the need for remote modules. Notably, this also introduces some security concerns into the app if not handled properly.

I think this can be removed, the app should still work.

Reply:
I think I added this while trying to resolve a bug with the dialog, then forgot to change it back later when I found the solution. I can delete it.